lib/event_rules.rb

```diff
@@ -0,0 +1,43 @@
+class EventRules
+
+  class << self
+
+    def get(event_name)
+      prefix = event_name.split('_').first
+      return find_rule(prefix)
+    end
+
+    private
+
+    def find_rule(prefix)
+      ruleset().each do |i, rule|
+        return rule if rule["PREFIX"].start_with?(prefix)
+      end
+
+      return nil
+    end
+
+    def ruleset()
```

Comment:
This rule set could be statically be initialised. This would mean these rules would only be parsed / checked on start up.
Env vars will be very cheap to iterate over but unnecessary. You might want to raise an error if there is a formatting issue, so parsing at start up is still a nice thing to do